test/e2e/e2e.helpers.js

```diff
@@ -1,8 +1,8 @@
 import { appendHTMLWithShadowRoots } from '../shadow-root-utils';
 
-export function setupTestWindow(done) {
+export function setupTestDocument(done) {
   cy.visit('./cypress/test-sandbox.html');
-  cy.window().then(done);
+  cy.document().then(done);
```

Comment:
I am not sure why, but even on the main branch, local E2E tests simply would not pass/run correctly with the existing `setupTestWindow`. [Since Cypress' docs seem to recommend `cy.document()` as a way to get a handle on the current window document](https://docs.cypress.io/api/commands/document#Yields), I used that instead of going through the window. 

(I'm significantly more familiar with Playwright, so I do not know if this is the most idiomatic way to do this)